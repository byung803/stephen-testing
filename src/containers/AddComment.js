import { connect } from 'react-redux';
import CommentBox from '../components/CommentBox';
import {saveComment} from '../actions'

export default connect(null, {saveComment})(CommentBox);