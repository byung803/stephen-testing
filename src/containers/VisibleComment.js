import { connect } from 'react-redux';
import CommentList from '../components/CommnetList';

const mapStateToProps = ({ comments }) => ({
    comments
});

export default connect(mapStateToProps)(CommentList);