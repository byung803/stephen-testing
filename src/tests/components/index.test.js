import React from 'react';
import { shallow } from 'enzyme';
import Index from '../../components';
import AddComment from '../../containers/AddComment';
import VisibleComment from '../../containers/VisibleComment';

let wrapper;

beforeEach(() => {
    wrapper = shallow(<Index />);
});

it('should have one AddComment Component', () => {
})

it('should have one VisibleComment Component', () => {
    expect(wrapper.find(VisibleComment).length).toEqual(1);
})

test('should take a snapshot of Index', () => {
    expect(wrapper).toMatchSnapshot();
});
