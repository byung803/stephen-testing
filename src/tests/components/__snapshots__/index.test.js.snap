// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should take a snapshot of Index 1`] = `
<div>
  <Connect(CommentBox) />
  <Connect(CommentList) />
</div>
`;
